src/router/index.js

```diff
@@ -68,11 +68,11 @@ const routes = [
     }
   },
   {
-    path: '/company-settings',
-    name: 'CompanySettings',
-    component: CompanySettings,
+    path: '/user-settings',
+    name: 'UserSettings',
+    component: UserSettings,
     meta: {
-      auth: true
+      auth: false
```

Comment:
i will make `auth` is `true`, if its okay for you?

Reply:
yes go for it